openslides/utils/autoupdate.py

```diff
@@ -40,7 +40,7 @@ def ws_add_site(message):
             # Skip apps that do not implement get_startup_elements
             continue
         for collection in get_startup_elements():
-            output.extend(collection.as_autoupdate_for_user(message.user))
+            output.extend(collection.as_autoupdate_for_user(message.user.id))
```

Comment:
Why is it not possible to use `message.user` instead of `message.user.id`? If we have to use the id then we should change the docstrings and argument names of all as_..._for_user(user.id) methods in utils/collection.py for better readablitity.